app/uk/gov/hmrc/mobilepayments/services/OpenBankingService.scala

```diff
@@ -39,6 +40,9 @@ class OpenBankingService @Inject() (
   connector:                                                         OpenBankingConnector,
   @Named("openBankingPaymentReturnUrl") openBankingPaymentReturnUrl: String) {
 
+  val appSimpleAssessment: String = "appSimpleAssessment"
+  val appSelfAssessment: String = "appSelfAssessment"
```

Comment:
Don't think these are being used? Not needed if they are as the enum holds the same values.